File: src/CallCheatOnline.Library.Data/ApiEntities/GetGameCheat.cs
using CallCheatOnline.Game.CallCheat.Models;
using CallCheatOnline.Library.Data.Utility;
using System;
using System.Collections.Generic;
using System.Text;

namespace CallCheatOnline.Library.Data.ApiEntities
{
	public class GetGameCheat
	{
		public bool CheatCallSuccessful { get; set; }
		public GetGameUser CheatCaller { get; set; }
		public GetGameUser LastClaimUser { get; set; }
		public List<GetGameClaim> PreCheatClaims { get; set; }

		public GetGameCheat(GameCheat gameCheat)
		{
			CheatCallSuccessful = gameCheat.CheatCallSuccessful;
			CheatCaller = new GetGameUser(gameCheat.CheatCaller);
			LastClaimUser = new GetGameUser(gameCheat.LastClaimUser);
			PreCheatClaims = gameCheat.PreCheatClaims.ApiGetGameClaims();
		}
	}
}

File: src/CallCheatOnline.Web.UI/src/models/rest/getgamecheat.ts
import { GetGameUser } from "./getgameuser";
import { GetGameClaim } from "./getgameclaim";

export class GetGameCheat 
{
    cheatCallSuccessful: boolean;
    cheatCaller: GetGameUser;
    lastClaimUser: GetGameUser;
    preCheatClaims: GetGameClaim[];
}